packages/api-client/src/connections/ApiConnection.ts

```diff
@@ -39,8 +40,9 @@ export class ApiConnection {
     endpoint: string,
     queryParameters?: QueryParameters | null,
     body?: RequestBody | null,
+    cookie?: string,
   ) {
-    return this.request('POST', endpoint, queryParameters, body);
+    return this.request('POST', endpoint, queryParameters, body, cookie);
```

Comment:
I'm not a fan of doing this.. It is only for pdf-export api, and we will remove it once we got pdf-export-server converted to a library.

Reply:
Will be removed by https://github.com/beyondessential/tupaia/pull/4087